glide.yaml

```diff
@@ -29,7 +29,7 @@ import:
   - certifiers/client
   - certifiers/files
 - package: github.com/tendermint/merkleeyes
-  version: unstable
+  version: origin/unstable
```

Comment:
It wasn't updating to origin/unstable with `glide update` - this fixed it. I'm not sure if this is a bug in glide or something else, but there were several people complaining about this issue in the glide repo.